src/Dynamic/Statistics/Statistics.tsx

```diff
@@ -0,0 +1,462 @@
+        const member = members.get(key as Snowflake);
+        if (isDefined(member)) {
+          data.push({ name: member.name, count: value });
+        }
+      });
+    }
+    data.sort((a, b) => (a.count < b.count ? 1 : -1));
+    return data;
+  };
+
+  const getPersonalMessageData = (): any[] => {
+    if (stats.isDefined()) {
+      const total = stats.get.messages.count;
+      const userCount = stats.get.messages.members[currentUser.id as string];
+      return [
+        { name: "me", value: userCount },
+        { name: "not me", value: total - userCount },
+      ];
+    }
+    return [];
+  };
+
+  const getArchitusMessageCount = (): number => {
+    if (stats.isDefined()) {
+      return stats.get.messages.members["448940980218101795"];
+    }
+    return 0;
+  };
+
+  return (
+    <Styled.PageOuter>
+      <h2>Statistics</h2>
+      <Styled.HeaderCards>
+        <Styled.MessageCard>
+          <Styled.ContentContainer>
+            <Styled.Icon name="comments" noAutoWidth />
+            <Styled.LabelContainer>
+              <Styled.CountUp end={getMessageCount()} duration={5} />
+              <Styled.Description>Messages Sent</Styled.Description>
+            </Styled.LabelContainer>
+          </Styled.ContentContainer>
+        </Styled.MessageCard>
+        <Styled.MemberCard>
+          <Styled.ContentContainer>
+            <Styled.Icon name="users" noAutoWidth />
+            <Styled.LabelContainer>
+              <Styled.CountUp end={getMemberCount()} duration={5} />
+              <Styled.Description>Members</Styled.Description>
+            </Styled.LabelContainer>
+          </Styled.ContentContainer>
+        </Styled.MemberCard>
+        <Styled.ArchitusCard>
+          <Styled.ContentContainer>
+            <Styled.Logo />
+            <Styled.LabelContainer>
+              <div>
+                <Styled.CountUp end={getArchitusMessageCount()} duration={5} />
+              </div>
+              <Styled.Description>Commands Executed</Styled.Description>
+            </Styled.LabelContainer>
+          </Styled.ContentContainer>
+        </Styled.ArchitusCard>
+      </Styled.HeaderCards>
+      <Styled.CardContainer>
+        <Styled.Card>
+          <Image src="https://cdn.discordapp.com/emojis/482104551902806016.png?v=1" />
+        </Styled.Card>
+        <Styled.Card>
+          <Image
+            src="https://cdn.discordapp.com/emojis/671530198352789524.png"
+            roundedCircle
+          />
+        </Styled.Card>
+        <Styled.Card>
+          <Styled.CountUp end={7} duration={5} />
+        </Styled.Card>
+        <Styled.Card>
+          <h5>Your Messages</h5>
+          <ResponsiveContainer>
+            <PieChart>
+              <Pie
+                data={getPersonalMessageData()}
+                cx={"50%"}
+                cy={"41%"}
+                innerRadius={"70%"}
+                outerRadius={"90%"}
+                fill="#844EA3"
```

Comment:
it would be nice if we used color theme tokens for all of the color values here as well